mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs

```diff
@@ -1096,7 +1096,7 @@ static bool ProcessComparisonsOverConstant(RuleProcessingContext context, Node n
 
             // Make sure that the terminal character of the pattern is a '%' character. Also
             // ensure that this character does not occur anywhere else. And finally, ensure
-            // that the pattern is atmost one character longer than the string itself
+            // that the pattern is almost one character longer than the string itself
```

Comment:
this should actually be `at most`, not almost.

Reply:
My bad. I fix it. Thanks for the review.